tests/basic_relay.go

```diff
@@ -63,15 +64,15 @@ func BasicRelay(network *network.LocalNetwork, teleporter utils.TeleporterTestIn
 	log.Info("Test Relaying from Subnet A to Subnet B")
 
 	log.Info("Starting the relayer")
-	relayerCleanup, readyChan := testUtils.RunRelayerExecutable(
+	relayerCleanup, readyChan /*readyChan*/ := testUtils.RunRelayerExecutable(
```

Comment:
might be a dumb question but what's the point of the in-line comment here?

Reply:
This was a test artifact. Removed now.